### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by tomschlenkhoff**
Distributed the fixes to the correct files.

### CODE REVIEW

File: sbin/update-ipsets.in [hunk 1]
```diff
@@ -4766,7 +4766,7 @@ update myip $[24*60] 0 ipv4 ip \
 update bogons $[24*60] 0 ipv4 both \
 	"http://www.team-cymru.org/Services/Bogons/bogon-bn-agg.txt" \
 	remove_comments \
-	"unroutable" \
+	"unrouteable" \
```
Comment (ktsaou) [inline sbin/update-ipsets.in]: oops! this will break the code. Please don't change this keyword.

Comment (ktsaou) [inline sbin/update-ipsets.in]: forget it. I think it is ok. I'll check it now.

**Comment by ktsaou** [comment]
As I see it, there is nothing in the code about categories. The only issue is that I might have to change by hand the category of the IP feeds on the site (they most probably get the new one, once the IP feed is updated).

I am merging this.
thanks @tomschlenkhoff !

**Comment by ktsaou** [comment]
ok, I updated the site too.
It seems that the only way for these texts to appear on the site, is to update the IP feeds. So I didn't change by hand anything. As time passes and the IP feeds get updated, the changes will be shown on the site.

**Comment by tomschlenkhoff** [comment]
This was quick. Thanks!

**Pull request merged**